src/api/java/com/minecolonies/api/research/effects/IResearchEffectManager.java

```diff
@@ -17,10 +17,32 @@
      */
     <W extends IResearchEffect<?>> W getEffect(final String id, @NotNull final Class<W> type);
 
+    /**
+     * Get the research effect strength which is assigned to a particular string
+     * or zero, if no matching effect is present.
+     *
+     * @param id   the id of the effect.
+     * @return the strength of the effect, or zero if it isn't present.
+     */
+     float getEffectValue(final String id);
```

Comment:
let's make those doubles and not floats, floats should be reserved to rendering logic